cuylerstuwe/chrome-extension-simple-udemy-section-scheduler
# #1: React Todo List

# PURPOSE

Resolves #1.

# METHOD

Implement a Todo List in the extension's popup.

# REVIEWER NOTES

Installation/testing Instructions:

1. Download repo locally.
2. Navigate a terminal to project directory.
3. Run `npm i` to install dependencies.
4. Run `npm run build:local-dev` to build extension in `./dist` subdirectory.
5. In a new Chrome tab, visit `chrome://extensions`.
6. In the upper-right corner of this page, make sure "Developer Mode" is turned on.
7. In the upper-left corner of this page, click "Load unpacked"; This will spawn a folder-picker.
8. Navigate the spawned folder-picker to select the `./dist` subdirectory generated in step 4.
9. Click the "puzzle piece" icon in the upper-right of Chrome's interface.
10. Click the "Schedule Udemy Course..." extension in the submenu.
11. A window should load that abides by the spec described in #1.
